src/shared/frame-rpc.js

```diff
@@ -29,6 +29,13 @@
 
 const VERSION = '1.0.0';
 
+/**
+ *  @constructor
+ *  @param {Window} src
+ *  @param {Window} dst
+ *  @param {string} origin
+ *  @param {(Object<string, ((any) => any)>) | ((any) => any)} methods
+ */
 export default function RPC(src, dst, origin, methods) {
```

Comment:
This is fine, although we don't actually make use of the non-object version of `methods` anywhere in the code, so it would be equally OK to just strip out that support.

Reply:
I'll take it out